### ISSUE TITLE
Accesibility or Simmetry?

### ISSUE THREAD

**Issue by SiriusFuenmayor** [filipinascimento/CVSymmetry#1]
If you take a look at the 'Makefile_Linux' file in the 'source' folder you can see that the ouput is named 'CVAccessibility'

`TARGET = CVAccessibility`

You can also see this in the output of the command

`#v "Backbone Accessibility h=2" n`

Is this a mistake or the linux tool is for calculating accesibility instead of simmetry?

**Comment by filipinascimento** [comment]
Hi, thanks for pointing this out. We have fixed the issue. At some point we used the name Normalized Accessibility to refer to symmetry. This is now corrected in the code and Makefiles. The calculations match the Backbone and Merged Symmetry of the paper.

Also, we have a new package that can be used to calculate symmetry in python:
https://github.com/ABenatti/network_symmetry

This may be preferable if you already have a Python-based pipeline.
